src/audio/kpb.c

```diff
@@ -522,7 +522,7 @@ static int kpb_prepare(struct comp_dev *dev)
 
 	if (!kpb->sel_sink || !kpb->host_sink) {
 		comp_info(dev, "kpb_prepare(): could not find sinks: sel_sink %d host_sink %d",
-			  (uint32_t)kpb->sel_sink, (uint32_t)kpb->host_sink);
+			  (uintptr_t)kpb->sel_sink, (uintptr_t)kpb->host_sink);
 		ret = -EIO;
```

Comment:
The tricky part with this approach is that when building for 64 bits, it will only print the lower 32 bits. But, I'm ok with the change.

Reply:
the only 64 bit target we have is the host AFAIK. And the host translates this to printf.